sdk/identity/Azure.Identity/src/MsalPublicClient.cs

```diff
@@ -147,12 +147,13 @@ public async ValueTask<AuthenticationResult> AcquireTokenInteractiveAsync(string
 
             AzureIdentityEventSource.Singleton.InteractiveAuthenticationExecutingInline();
 
-            var result = await AcquireTokenInteractiveCoreAsync(scopes, claims, prompt, loginHint, tenantId, async, cancellationToken).ConfigureAwait(false);
+            var result = await AcquireTokenInteractiveCoreAsync(scopes, claims, prompt, loginHint, tenantId, useEmbeddedWebView, systemWebViewOptions, embeddedWebViewOptions, async, cancellationToken).ConfigureAwait(false);
             LogAccountDetails(result);
             return result;
         }
 
-        protected virtual async ValueTask<AuthenticationResult> AcquireTokenInteractiveCoreAsync(string[] scopes, string claims, Prompt prompt, string loginHint, string tenantId, bool async, CancellationToken cancellationToken)
+        protected virtual async ValueTask<AuthenticationResult> AcquireTokenInteractiveCoreAsync(string[] scopes, string claims, Prompt prompt, string loginHint, string tenantId, bool? useEmbeddedWebView, SystemWebViewOptions systemWebViewOptions, EmbeddedWebViewOptions embeddedWebViewOptions,
```

Comment:
Can we pass the whole `BrowerCustomizationOptions` here instead of all its discrete properties?